generator/sbpg/targets/resources/sbp_construct_template.py.j2

```diff
@@ -14,15 +14,25 @@
 (((description|comment_links)))
 """
 
+((* if msgs|has_real_message -*))
 import json
 
 import construct
 
 from sbp.msg import SBP, SENDER_ID
 from sbp.utils import fmt_repr, exclude_fields, walk_json_dict, containerize
+((*- else -*))
+import construct
+
+from sbp.utils import fmt_repr
+((*- endif -*))
 
 ((*- for i in include *))
+  ((*- if i in include_map *))
+from (((module_path))).(((i))) import ((* for name in include_map[i] -*))((( name )))((*- if not loop.last *)), ((* endif -*))((*- endfor *))  # noqa
```

Comment:
ideally remove the `noqa` and post-process the generated python through autoflake to remove the unused imports.